pkgs/top-level/python-packages.nix

```diff
@@ -19123,6 +19123,29 @@ let
     };
   };
 
+  cmdtest = buildPythonPackage rec {
+    name = "cmdtest-${version}";
+    version = "0.15";
+    disabled = isPy3k || isPyPy;
+
+    buildInputs = with self; [ cliapp ttystatus ];
+    propagatedBuildInputs = with self; [ cliapp ttystatus markdown ];
+
+    # TODO: cmdtest tests must be run before the buildPhase
+    doCheck = false;
+
+    src = pkgs.fetchgit {
```

Comment:
for availability reasons I guess? [1]
also, apologies, 0.15 appeared to be the latest release when I checked, 0.18 was made about 2 hours ago. I can easily update this.

[1] https://nixos.org/nixpkgs/manual/#sec-sources